roles/installer/tasks/database_configuration.yml

```diff
@@ -14,14 +14,26 @@
     name: '{{ meta.name }}-postgres-configuration'
   register: _default_pg_config_resources
 
-- name: Check for old PostgreSQL configuration secret
+- name: Check for specified old PostgreSQL configuration secret
   k8s_info:
     kind: Secret
     namespace: '{{ meta.namespace }}'
     name: '{{ tower_old_postgres_configuration_secret }}'
-  register: old_pg_config
+  register: _custom_old_pg_config_resources
   when: tower_old_postgres_configuration_secret | length
 
+- name: Check for default old PostgreSQL configuration
+  k8s_info:
+    kind: Secret
+    namespace: '{{ meta.namespace }}'
+    name: '{{ meta.name }}-old-postgres-configuration'
+  register: _default_old_pg_config_resources
```

Comment:
This makes sense.  In case the user doesn't explicitly specify the name of the old postgres configuration secret, if it is named as recommended in the docs, it will still be picked up by this task.